packages/medusa/src/strategies/batch-jobs/product/types/index.ts

```diff
@@ -59,7 +59,7 @@ export type ProductExportPriceData = {
   region?: { name: string; currency_code: string; id: string }
 }
 
-export type ProductExportColumnSchemaEntity = "product" | "variant"
+export type ProductExportColumnSchemaEntity = "product" | "variant" | "productAndVariant"
```

Comment:
Currently, each of the accessors is responsible for computing a Medusa entity, either `Product` or `Variant`. Introducing a new type of accessor doesn't make sense since it would have an overlapping responsibility with already existing accessor types.